src/steamship/client/client.py

```diff
@@ -129,3 +149,18 @@ def get_space(self) -> Space:
             )
         logging.info(f"Got space: {space.data.id}")
         return space.data
+
+    def list_comments(
```

Comment:
Since `comments` are a sub-item of `Task`, and this requires a `task_id`, wouldn't it make sense to put this on the `Task` object?